public/application.js

```diff
@@ -15,62 +14,45 @@ function humanVersusComputer() {
         dataType: "json"
     });
     request.then((response) => {
-        computerTurn(response);
-        makeHumanMove(response);
-      });
+      putMarkersDown(response);
+      if (gameIsOver(response)) {
+        gameResult(response);
+      };
+    });
   });
 };
 
-function makeHumanMove(response) {
-  if (response.valid === true) {
-    $('#' + response.spot).empty();
-    $('#' + response.spot).append(response.marker);
-    endGame(response);
+function gameIsOver(response) {
```

Comment:
is this function necessary? if we rename response object to something like `response.isGameOver` we wouldn't need this function.